src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -12,6 +12,36 @@ public partial class ArrayConverter : System.ComponentModel.CollectionConverter
     {
         public ArrayConverter() { }
         public override object ConvertTo(System.ComponentModel.ITypeDescriptorContext context, System.Globalization.CultureInfo culture, object value, System.Type destinationType) { return default(object); }
+        public override System.ComponentModel.PropertyDescriptorCollection GetProperties(System.ComponentModel.ITypeDescriptorContext context, object value, System.Attribute[] attributes) { return default(System.ComponentModel.PropertyDescriptorCollection); }
+        public override bool GetPropertiesSupported(System.ComponentModel.ITypeDescriptorContext context) { return default(bool); }
+    }
+    public partial class AttributeCollection : System.Collections.ICollection, System.Collections.IEnumerable
+    {
+        public static readonly System.ComponentModel.AttributeCollection Empty;
+        protected AttributeCollection() { }
+        public AttributeCollection(params System.Attribute[] attributes) { }
+        protected virtual System.Attribute[] Attributes { get; }
+        public bool Contains(System.Attribute attribute) { return default(bool); }
+        public bool Contains(System.Attribute[] attributes) { return default(bool); }
+        public void CopyTo(System.Array array, int index) { }
```

Comment:
I follow the desktop framework surface here because we're adding these APIs purely for compatibility.